20180125/AGENDA.md

```diff
@@ -0,0 +1,11 @@
+# Sardana Follow-up Meeting - 2018/01/25
+
+To be held on 2018/01/25 at 14:00
+
+# Agenda
+1. Presentation ... ?
```

Comment:
I could prepare some slides regarding the taurus3 and taurus4 URI names and the necessary changes in sardana. This could kick off sardana-org/sardana#668. Yes, I know.... this is not very interesting topic:)

Reply:
Apart of this one, I can prepare some demo of the use of Sardana with Elastic stack (Logstash, Elasticsearch and Kibana). The sardana implementation is my personal [branch](https://github.com/reszelaz/sardana/tree/logstash-log-handler) and necessary configs of the Elastic in this [repository](https://github.com/reszelaz/sardana-elastic). I will open a PR soon for that. This is certaunly more cool than the previous proposal. But maybe we could do both..

Reply:
For me both are interesting, maybe more interesting the second one, but it could be nice to have some info about both topics. It doesn't need to be a fancy presentation, it's OK with a small explanation and if we want/need more details we can continue in another moment/channel.

Reply:
@amilan thanks for the feedback! Since there is no more time I already reflected these presentations in the Agenda. From the same time constraints IMHO the agenda is already defined:) So I'm merging this PR.